sambacc/addc.py

```diff
@@ -215,3 +217,24 @@ def _group_add_members_cmd(group_name: str, members: list[str]) -> list[str]:
         ",".join(members),
     ].argv()
     return cmd
+
+
+def _ifnames() -> list[str]:
+    import socket
```

Comment:
Why local import instead top-level one? 
(Not an issue, but wondering why have it like this).

Reply:
I figured it would save an import at start-up time. Only if the interface filtering gets used would would the module get imported. It's probably a minuscule difference though. :-)